colour/models/rgb/dataset/eci_rgb_v2.py

```diff
@@ -88,55 +87,27 @@
 """
 
 
-def _eci_rgb_v2_OECF(value):
+def _scale_domain_0_100_range_0_1(value, callable_):
     """
-    Defines the *ECI RGB v2* colourspace opto-electronic conversion function.
+    Scales the domain of given callable to [0, 100] and its range to [0, 1].
```

Comment:
Maybe this docstring should further specify the signature of the callable?
Right now it implicitly assumes a single numeric object (or ndarray) as input and output.